pkg/realmd/operation.html

```diff
@@ -8,6 +8,15 @@ <h4 class="modal-title realms-op-title"></h4>
         <div class="realms-op-fields">
           <p class="realms-op-leave-only-row"></p>
           <table class="form-table-ct realms-op-form">
+            <tr class="realms-op-error" hidden>
+              <td colspan="3">
+                <div class="alert alert-danger">
+                  <span class="pficon pficon-error-circle-o"></span>
+                  <strong class="realms-op-message"></strong>
+                </div>
+                <pre class="realms-op-diagnostics" hidden></pre>
+              </td>
+            </tr>
```

Comment:
I'm fine moving to the bottom where it was before, if that's the convention. Patternfly, I think, has form submission errors at the top with errors that are tied to specific form fields placed inline.
http://www.patternfly.org/pattern-library/forms-and-controls/errors-and-validation/